docs/docs/ref-01-top-level-api.md

```diff
@@ -161,6 +161,10 @@ If the optional callback is provided, it will be executed after the component is
 > `ReactDOM.render()` does not modify the container node (only modifies the children of the container). In
 > the future, it may be possible to insert a component to an existing DOM node without overwriting
 > the existing children.
+>
+> `ReactDOM.render()` currently returns a reference to the root ReactComponent instance.  However, using this return value is legacy
+> and should be avoided.  If you need a reference to the root ReactComponent instance, the preferred solution is to attach a
+> [callback ref](/react/docs/more-about-refs.html#the-ref-callback-attribute) to the root element.
```

Comment:
What's the difference between a callback ref and the value provided to the optional callback function to ReactDOM.render?